charts/karpenter/templates/default_provisioner.yaml

```diff
@@ -3,6 +3,9 @@ apiVersion: karpenter.sh/v1alpha5
 kind: Provisioner
 metadata:
   name: default
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/resource-policy": keep
```

Comment:
What do these do?

Reply:
this tells helm that it's a post-install hook and to keep the resource rather than deleting it right after applying.